Source/Core/Core/HW/WiimoteReal/IONix.cpp

```diff
@@ -84,9 +83,15 @@ void WiimoteScanner::FindWiimotes(std::vector<Wiimote*> & found_wiimotes, Wiimot
 	inquiry_info scan_infos[max_infos] = {};
 	auto* scan_infos_ptr = scan_infos;
 	found_board = nullptr;
+	// query in limited IAC - liac - using code from hcitool
+	// utils/tools/hcitool.c
+	const int liac_value = 0x9e8b00; // LIAC (https://www.bluetooth.org/en-us/specification/assigned-numbers/baseband)
```

Comment:
Nit: I would just inline that as const uint8_t lap[3] = { 0x00, 0x8b, 0x9e };

I think the comment about this coming from hcitool can be removed given that this is basically a documented constant. However I'd like a comment that explains why this code is using limited IAC (basically a short version of your PR description).